### DISCUSSION TITLE
Still hiring?

### DISCUSSION THREAD

**Discussion by binaryben** [directus/directus#16497] [Archive]
Hello, the [README.md on the project repo](https://github.com/directus/directus#readme) links to [this missing discussion about job opportunities](https://github.com/directus/directus/discussions/15048).

I was hoping to apply, but wondering if the missing discussion meant the roles were no longer available? Specifically I was looking for it again to send an to email @rijkvanzanten with some questions as I think he had his email address posted there for that purpose.

**Comment by br41nslug** [comment]
You can apply here 😄 https://directus.io/careers/

**Comment by binaryben** [comment]
Thanks @br41nslug, but the [lever.co](https://jobs.lever.co/directus.io/6296d851-cd60-40d6-b9e5-f2015502f932/apply) link the job posting directs to when going to apply is dead as well. So not hiring anymore software engineers?

**Reply by br41nslug** [reply]
Those were taken down intentionally, sorry

**Reply by binaryben** [reply]
That's alright, but just trying to confirm if that means you're no longer hiring for those roles?

EDIT: I see they are no longer listed on the careers page you linked earlier now, so I guess that is the answer. Bugger!
